apps/src/templates/export/expo/app.json.ejs

```diff
@@ -14,6 +14,9 @@
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },
+    "updates": {
+      "fallbackToCacheTimeout": 0
+    },
```

Comment:
What does this do?

Reply:
I generated a new project with a newer SDK and it included this by default. It is their recommended value to ensure that app updates occur asynchronously (impacted the next launch of the app)